components/eventing-controller/README.md

```diff
@@ -33,6 +45,16 @@ This section explains how to use the Eventing Controller.
     | reconcile-period        | The period between triggering of reconciling calls. | 10 minutes    |
     | enable-debug-logs       | Enable debug logs.                                  | false         |
 
+- The Nats Controller comes with the following command line argument flags:
+
+    | Flag                    | Description                                         | Default Value |
+    | ----------------------- | --------------------------------------------------- | ------------- |
+    | metrics-addr            | The address the metric endpoint binds to.           | :8080         |
+    | reconcile-period        | The period between triggering of reconciling calls. | 10 minutes    |
+    | enable-debug-logs       | Enable debug logs.                                  | false         |
+    | max-reconnects          | Maximum number of reconnect attempts.               | 10            |
```

Comment:
```suggestion
    | `max-reconnects`          | The maximum number of reconnection attempts.               | `10 `           |
```

Reply:
I added `the` just to be consistent with other descriptions. If you prefer to remove it, let's do it at the beginning of all descriptions.